server/pbenchinacan/load_keycloak.sh

```diff
@@ -118,7 +119,7 @@ CLIENT_CONF=$(curl -si -f -X POST \
        "serviceAccountsEnabled": true,
        "enabled": true,
        "attributes": {"post.logout.redirect.uris": "'${KEYCLOAK_REDIRECT_URI}'"},
-       "redirectUris": ["'${KEYCLOAK_REDIRECT_URI}'"]}')
+       "redirectUris": ["'${KEYCLOAK_REDIRECT_URI}'", "'${KEYCLOAK_DEV_ORIGIN}/*'" ]}')
```

Comment:
Do we need/want `KEYCLOAK_DEV_ORIGIN` in the `"post.logout.redirect.uris"` list?

Reply:
Well, in theory, yeah; I deferred it because I couldn't figure out *how* because despite the plural in the key it doesn't actually take a *list*. And then I pushed an update before I remembered that. 😦 

I believe I've discovered that using `"+"` actually isn't in this case taken as an undesired "universal" but rather as a deferral to the normal `redirectUris` list; and as such ought to work.

If only I could test it ...